lib/swagger_final_path_processing.py

```diff
@@ -33,8 +33,19 @@ def process_output(
                 'version': '2.0.0'},
             'host': '<vcenter>',
             'securityDefinitions': {
+                "api_key": {
```

Comment:
I think the description said session_id, didn't it? Let's use session_id, as the well-known meaning of api_key is somewhat different (an ID assigned to the application by some side channel, e.g. registration, which is usually long living, i.e months).

Reply:
Renamed as suggested - session_id. Also removed the global security definition and added session_id security per operation (where necessary).